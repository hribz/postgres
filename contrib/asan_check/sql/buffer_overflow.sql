CREATE EXTENSION asan_check;
SELECT buffer_overflow();
