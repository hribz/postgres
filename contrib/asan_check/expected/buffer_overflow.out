CREATE EXTENSION asan_check;
ERROR:  extension "asan_check" already exists
SELECT buffer_overflow();
server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
connection to server was lost
